### DISCUSSION TITLE
Unable to login after 24 hrs when deployed on heroku, needs re registeration.

### DISCUSSION THREAD

**Discussion by kunalgaurav4** [strapi/strapi#7588] [Legacy Topics]
Deployed my blog project on Heroku using postgres database. But I need to re register after 24 hrs and the data is also lost.
It should not lose data.I am not sure if I require changes to the configration.

**Comment by derrickmehaffy** [comment]
Hello!

It doesn't sound like Strapi is actually using PostgreSQL but is using a SQLite database (which is why you would loose your data, along with this FAQ entry: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#why-is-my-app-s-database-and-uploads-resetting-on-heroku )

Can you please double check your `./config/database.js` and `./config/env/production/database.js` (this 2nd one might not exist) to ensure the file is setup to properly load environment variables to connect to a PostgreSQL database, you can find the structure here: https://strapi.io/documentation/v3.x/concepts/configurations.html#example

**Reply by kunalgaurav4** [reply]
Yes it works. Documentations are really good.
